src/postcss/main.css

```diff
@@ -50,6 +50,7 @@
 @import './pages/accordions.css';
 @import './pages/tables.css';
 @import './pages/modals.css';
+@import './pages/calendar-page.css';
```

Comment:
It's unnecessary to add the `-page` prefix to the file name since it's already in the `pages` folder so has the context :)